### PR TITLE
New blog post on Swagger UI

### PR DESCRIPTION
**Pull request by phillip-kruger**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

File: _posts/2020-11-25-stylish-api.adoc [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    font-size:70%;
+    text-align: center;
+}
+----
+
+<1> here set the `topbar` background color.
+
+image::acme_css.png[acme css]
+
+You can change any styling element in this css file, you need to place this file called `style.css` in `src/main/resources/META-INF/branding`.
```
Comment (gsmet) [inline _posts/2020-11-25-stylish-api.adoc:109]: Same here about the name being very generic.

File: _posts/2020-11-25-stylish-api.adoc [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+You can also set the HTML title, and add a footer:
+
+[source,properties]
+----
+quarkus.swagger-ui.title=ACME API
+quarkus.swagger-ui.footer=&#169; 2020 . ACME
+----
+
+Along with other OpenAPI Header fields that can be set via config (as discussed in link:/blog/openapi-for-everyone/[this post]):
```
Comment (gsmet) [inline _posts/2020-11-25-stylish-api.adoc:121]:
```suggestion
Along with other OpenAPI Header fields that can be set via configuration properties (as discussed in link:/blog/openapi-for-everyone/[this post]):
```

File: _posts/2020-11-25-stylish-api.adoc [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+mp.openapi.extensions.smallrye.info.license.url=http://www.apache.org/licenses/LICENSE-2.0.html
+----
+
+The UI is now fully branded:
+
+image::acme_footer.png[acme footer]
+
+== Other Swagger UI Options
+
+Another new feature available in Quarkus (v1.10.0 +) is the ability to set any of the https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/[configuration options] available in Swagger UI. As an example, we can set the `urls` and add the petstore (as the default selected option) to Swagger UI:
```
Comment (gsmet) [inline _posts/2020-11-25-stylish-api.adoc:141]:
```suggestion
Another new feature available in Quarkus (version 1.10+) is the ability to set any of the https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/[configuration options] available in Swagger UI. As an example, we can set the `urls` and add the petstore (as the default selected option) to Swagger UI:
```

File: _posts/2020-11-25-stylish-api.adoc [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+[source,properties]
+----
+quarkus.swagger-ui.supported-submit-methods=get
+----
+
+Note below the missing `Try it out` button on the `POST`
+
+image::tryitout.png[try it out]
+
+All other Swagger UI options are now available to configure the UI.
```
Comment (gsmet) [inline _posts/2020-11-25-stylish-api.adoc:165]:
```suggestion
All the other Swagger UI options are now available to configure the UI.
```

File: _posts/2020-11-25-stylish-api.adoc [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+== Other small new features
+
+Two small new features in OpenAPI and Swagger UI, the ability to add the Health Endpoints and the ability to disable the UI and/or Schema in Runtime.
```
Comment (gsmet) [inline _posts/2020-11-25-stylish-api.adoc:169]:
```suggestion
Let me present you two small new features in OpenAPI and Swagger UI: the ability to add the Health Endpoints and the ability to disable the UI and/or Schema at runtime.
```

File: _posts/2020-11-25-stylish-api.adoc [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+If you are using the `smallrye-health` extension, you can add the Health Endpoints to OpenAPI:
+
+[source,properties]
+----
+quarkus.health.openapi.included=true
+----
+
+image::health.png[health]
+
+=== Disable in Runtime
```
Comment (gsmet) [inline _posts/2020-11-25-stylish-api.adoc:182]:
```suggestion
=== Disable at Runtime
```

File: _posts/2020-11-25-stylish-api.adoc [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+
+If you included the UI in your app (`quarkus.swagger-ui.always-include=true`), you can now disable it when starting the application.
+
+----
```
Comment (gsmet) [inline _posts/2020-11-25-stylish-api.adoc:186]:
```suggestion
[source]
----
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Nice work. Added some suggestions.

File: _posts/2020-11-25-stylish-api.adoc [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+
+=== Logo
+
+As part of the custom branding, you can supply your own logo to replace the Quarkus logo. For example, let say you own company that makes everything, ACME, and you are using REST Services for your online store, and wants to brand the Swagger UI Page:
+
+image::acme_logo.png[acme logo]
+
+NOTE: Hot reload is not working for logo changes, and remember browser cache, you might need to https://refreshyourcache.com/en/cache/[force refresh] your browser.
+
+To supply your own logo, you need to place a file called `logo.png` in `src/main/resources/META-INF/branding`.
```
[the remark this answers is not in the document]
Comment (phillip-kruger) [reply _posts/2020-11-25-stylish-api.adoc:66]: I have no strong feeling on this. We can change this.

[the remark this answers is not in the document]
Comment (gsmet) [reply _posts/2020-11-25-stylish-api.adoc:66]: What I don't like is that there's a good chance the logo version used here might be a bit specific to these UIs and `branding/logo.png` is typically where I would put a very generic logo.

Now we would need a name to cover all these UIs.

[the remark this answers is not in the document]
Comment (phillip-kruger) [reply _posts/2020-11-25-stylish-api.adoc:66]: This logo will be used for all UIs.
What we can do later is allow a more specific logo per UI, calling it something like "swagger-ui.png" or "graphql-ui.png" or something like that. 
The style.css already works like that. (I did not mention that in the blog post)

[the remark this answers is not in the document]
Comment (phillip-kruger) [reply _posts/2020-11-25-stylish-api.adoc:66]: So with style, it will use the specific one (per UI) or fallback to style.css. We can do the same for logo.

**Review by gsmet — APPROVED** [review]

**Pull request merged**